zerver/tests/fixtures/slack_message_conversion.json

```diff
@@ -84,6 +84,11 @@
       "name": "italic_and_strike_conversion",
       "input": "_~italic~_ and ~_strike_~",
       "conversion_output": "*~~italic~~* and ~~*strike*~~"
+    },
+    {
+      "name": "new_line_test",
+      "input": "\n*abc*\n",
+      "conversion_output": "\n**abc**\n"
```

Comment:
I might be more explicit in the commit message that this output would have been `"\n*abc*\n"` prior to these changes.

Reply:
hmm here's how I've rephrased the commit description, does that look clearer?
```
Previously, the regex in `slack_message_conversion.py` couldn't capture
and reformat strings surrounded by newlines, such as "\n*ABC*\n". This
kind of string is likely not uncommon to be generated by both the Slack
exporter itself or by our Slack message block and attachment formatters
(`render_blocks` and `render_attachments`).
```